package.json

```diff
@@ -18,15 +18,15 @@
     "build:dev": "npm run clean && tsc -p . && webpack",
     "lint": "eslint -c .eslintrc.js --ext .ts src",
     "lint:fix": "npm run lint -- --fix",
-    "test": "jest --collectCoverage",
+    "test": "npm run lint && jest --collectCoverage",
```

Comment:
should this be `npm run lint && npm run test:unit`?

Reply:
Yes, thanks for catching that. I'll switch it out